### DISCUSSION TITLE
pages build and deployment problem

### DISCUSSION THREAD

**Discussion by ghost** [greenelab/lab-website-template#201] [Q&A] [answered]
不好意思，我的英文不好，所以用中文發問
![AA](https://github.com/greenelab/lab-website-template/assets/137671359/79951f9d-abe3-4ee3-b141-85fd133b1713)
如第一張圖，在這個步驟出現問題後，接下來 "pages build and deployment" 的 "deploy" 都處於 "Waiting" 狀態，想請問有什麼方法可以解決，謝謝
![BB](https://github.com/greenelab/lab-website-template/assets/137671359/13dcb2bc-cd1f-4d4c-ae76-f373aaded66f)
![CC](https://github.com/greenelab/lab-website-template/assets/137671359/1c49ad68-56ef-4375-b916-05590911ca54)

**Comment by vincerubinetti** [comment, marked as answer]
I'm guessing this is related to the [massive outage GitHub had earlier](https://www.githubstatus.com/incidents/gqx5l06jjxhp). It looks like your [page builds](https://github.com/ntou-lcl/ntou-lcl.github.io/actions) have been successful since then.
